src/Ninja/G1CP/Content/Fixes/Gamesave/fix055_ReactivateInExtremo.d

```diff
@@ -0,0 +1,594 @@
+    if (val == 0)                                          { good = FALSE; }; addr += 4;
+    if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)            { good = FALSE; }; addr += 1;
+    if (MEM_ReadInt( addr) != varPlyingId)                 { good = FALSE; }; addr += 4;
+    if (MEM_ReadByte(addr) != zPAR_OP_IS)                  { good = FALSE; }; addr += 1;
+    if (MEM_ReadByte(addr) != zPAR_TOK_RET)                { good = FALSE; };
+    if (!good) {
+        MEM_Info("Content of function 'B_InExtremoStartMusic' not as expected");
+        return FALSE;
+    };
+
+    // Check that the content of "B_InExtremoStopMusic" is as expected
+    var int fncUnSndTrgId; fncUnSndTrgId = MEM_GetFuncID(Wld_SendUnTrigger);
+    addr = G1CP_GetFuncStart(fncStopMsId);
+    good = TRUE;
+    if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)            { good = FALSE; }; addr += 5;
+    if (MEM_ReadByte(addr) != zPAR_TOK_CALLEXTERN)         { good = FALSE; }; addr += 1;
+    if (MEM_ReadInt( addr) != fncUnSndTrgId)               { good = FALSE; }; addr += 4;
+    if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)            { good = FALSE; }; addr += 5;
+    if (MEM_ReadByte(addr) != zPAR_TOK_CALLEXTERN)         { good = FALSE; }; addr += 1;
+    if (MEM_ReadInt( addr) != fncUnSndTrgId)               { good = FALSE; }; addr += 4;
+    val = MEM_ReadInt(addr+1);
+    if (MEM_ReadByte(addr) == zPAR_TOK_PUSHVAR) {
+        val = G1CP_GetIntI(val, 0, 1);
+    } else if (MEM_ReadByte(addr) != zPAR_TOK_PUSHINT)     { good = FALSE; }; addr += 1;
+    if (val != 0)                                          { good = FALSE; }; addr += 4;
+    if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)            { good = FALSE; }; addr += 1;
+    if (MEM_ReadInt( addr) != varPlyingId)                 { good = FALSE; }; addr += 4;
+    if (MEM_ReadByte(addr) != zPAR_OP_IS)                  { good = FALSE; }; addr += 1;
+    if (MEM_ReadByte(addr) != zPAR_TOK_RET)                { good = FALSE; };
+    if (!good) {
+        MEM_Info("Content of function 'B_InExtremoStopMusic' not as expected");
+        return FALSE;
+    };
+
+    // Check that the content of "DIA_Grim_INEXTREMO_Info" is as expected: Change of routine
+    bytes[0] = zPAR_TOK_CALLEXTERN<<24;
+    bytes[1] = MEM_GetFuncID(Npc_ExchangeRoutine);
+    matches = G1CP_FindInFunc(fncDiaGrimId, _@(bytes)+3, 5);
+    count = MEM_ArraySize(matches);
+    if (count == 1) {
+        addr = MEM_ArrayRead(matches, 0);
+    };
+    MEM_ArrayFree(matches);
+    if (count != 1) {
+        MEM_Info("Content of function 'DIA_Grim_INEXTREMO_Info' not as expected (no 'Npc_ExchangeRoutine')");
+        return FALSE;
+    };
+    val = MEM_ReadInt(addr-4);
+    if (!Hlp_StrCmp(G1CP_GetStringI(val, 0, ""), "InExtremo")) {
+        MEM_Info("Content of function 'DIA_Grim_INEXTREMO_Info' not as expected (no 'InExtremo' routine)");
+        return FALSE;
+    };
+
+    // Check that the function "B_InsertInExtremo" is not called anywhere
```

Comment:
In between here, also verify the content of the function `IE_397_Announcer_Announce_Condition`. It has to be either:
```D
    if (Kapitel == 2) {
        return TRUE; // Or anything non-zero (literal or constant)
    };
```
or
```D
    if (Kapitel == 2) {
    };
```
Check the end of the function with `zPAR_TOK_RETURN` to make sure there is nothing else added.

In the latter case, overwrite the function with a function looking like the first code block.